Update exception replay sensitive data setup (#26160)
DataDog/documentation: content/en/error_tracking/backend/exception_replay.md

### Redacting sensitive data

> Once you enable Sensitive Data Scrubbing, by default, variable data linked to specific identifiers deemed sensitive, such as `password` and `accessToken`, are automatically redacted. Enable Sensitive Data Scrubbing rules [in-app][7]. See the full [list of redacted identifiers][1].

Reviewer: ```suggestion
After you enable Sensitive Data Scrubbing, by default, variable data linked to specific identifiers deemed sensitive, such as `password` and `accessToken`, is automatically redacted. Enable Sensitive Data Scrubbing rules [in Datadog][7]. See the full [list of redacted identifiers][1].
```

Reviewer: Per the [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#words-and-phrases)
